### ISSUE TITLE
Python 2.7 compat

### ISSUE THREAD

**Issue by bcipolli** [johnchoiniere/pfx_parser#1]
I can confirm this script chokes on Python 2.7; at least because of the use of `urllib.request`. Using `six` could alleviate this

**Comment by johnchoiniere** [comment]
I wrote this for 3.4; I can try to add 2.7 compatibility, though - I haven't thought about that at all up to this point.

**Comment by bcipolli** [comment]
Yep. You expressed doubt in the article about it; I just thought I'd follow up :)
